File: decompiled/sources/com/hjq/http/exception/ResultException.java
package com.hjq.http.exception;

public final class ResultException extends HttpException {
    private final Object mData;

    public ResultException(String str, Object obj) {
        super(str);
        this.mData = obj;
    }

    public ResultException(String str, Throwable th, Object obj) {
        super(str, th);
        this.mData = obj;
    }

    public Object getData() {
        return this.mData;
    }
}

File: decompiled/sources/com/hjq/http/exception/ResultException.kt
package com.hjq.http.exception

class ResultException : HttpException {
    private val mData: Any

    constructor(str: String, obj: Any) : super(str) {
        mData = obj
    }

    constructor(str: String, th: Throwable, obj: Any) : super(str, th) {
        mData = obj
    }

    fun getData(): Any {
        return mData
    }
}
